backend/src/services/model.ts

```diff
@@ -319,6 +322,18 @@ export async function updateModel(user: UserInterface, modelId: string, modelDif
   if (modelDiff.settings?.mirror?.destinationModelId && modelDiff.settings?.mirror?.sourceModelId) {
     throw BadReq('You cannot select both mirror settings simultaneously.')
   }
+  if (modelDiff.collaborators) {
```

Comment:
Can you pull this functionality out into its own function? It would be useful to reuse this logic in create model as well